.github/stale.yml

```diff
@@ -0,0 +1,22 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - Never Stale
+# Label to use when marking an issue as stale
+staleLabel: Stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed in 7 days if no further activity occurs. 
+  If you feel this is incorrect please comment to keep it alive, with a reason
+  why. 
+   
+  To prevent long-term issues (e.g. Planning) closing add the 
```

Comment:
This final sentence doesn't quite parse well for me. Could we adjust to something like:
  
"To prevent closure, e.g. for long-term planning issues, add the "Never Stale" label.